src/helm/config/model_deployments.yaml

```diff
@@ -1766,4 +1766,18 @@ model_deployments:
     client_spec:
       class_name: "helm.clients.together_client.TogetherClient"
       args:
-        together_model: Qwen/Qwen1.5-72B
\ No newline at end of file
+        together_model: Qwen/Qwen1.5-72B
+
+  - name: Qwen/Qwen-VL
```

Comment:
it seems like this is the huggingface implementation, so should we not call this `huggignface/Qwen-VL`